### ISSUE TITLE
[error] Failed to authenticate on Console Websocket

### ISSUE THREAD

**Issue by Dian8** [edgurgel/poxa#125]
Sometimes auth to built-in console works fine, But sometimes there is a following unspecified error :
[error] Failed to authenticate on Console Websocket
Why does it happen? How to log in?

**Comment by edgurgel** [comment]
🤔 I will double check but it should be very similar to Pusher's auth etc.

**Comment by Dian8** [comment]
I've found it happens only when authentication from remote host. From localhost auth works fine

**Comment by Dian8** [comment]
This issue seems to be back (

I have virtual machine with docker. 
Docker is runned as follows:
`docker run --rm -p 192.168.1.150:8080:8080 -v $PWD/mypoxa.conf:/app/poxa/running-config/poxa.conf edgurgel/poxa-automated`

Config is default.

When I pulled new docker image (latest) everything worked fine. 
After reboot authentication works only from this vm. Auth from any other machine results "Authentication error" (for correct app_key and secret)

**Comment by edgurgel** [comment]
yeah I'm wondering if this is a websocket issue. Websocket is really annoying with hostnames. I will investigate more 👍

**Comment by Dian8** [comment]
I've found a following may help **sometimes** (~50%) for some time:

```
docker kill <poxa_container>
docker system prune -a
reboot
start poxa container
```
